.github/workflows/cmake.yml

```diff
@@ -32,6 +32,19 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      # Pull requests can trail behind `master` and GitHub only updates the merge ref when the PR branch is updated by the author,
```

Comment:
> and GitHub only updates the merge ref when the PR branch is updated by the author,

Just to document this here again, it also does so when the target branch is updated, but the default settings for fetching apparently don't pick that up.

The documentation for `actions/checkout` says:

> When checking out the repository that triggered a workflow, this defaults to the reference or SHA for that event.

Since it doesn't magically work when just rerunning the CI, I presume that it uses the SHA.

Reply:
Good point - rewrote that comment.